CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.6.20 (November 21, 2019)
+
+### Fixed
+- Use default IOCP concurrency value (#1161).
+- setting FD_CLOEXEC in pipe (#1095).
```

Comment:
inconsistent capitalization w/ previous line (do we care?)

Reply:
I'll try to fix it later... need to get this out.